CODEOWNERS

```diff
@@ -30,29 +30,43 @@ NOTICES     @garyverhaegen-da @aherrmann-da @cocreature @dasormeter
 /daml-lf/ @remyhaemmerle-da @cocreature
 /daml-lf/data-scalacheck/ @S11001001
 /daml-lf/governance.rst @sofiafaro-da
-/daml-lf/kv-transaction-support/ @gerolf-da @miklos-da
+/daml-lf/kv-transaction-support/ @digital-asset/kv-committer
 /daml-lf/transaction-test-lib/ @S11001001
 
 /ledger-service/    @S11001001 @cocreature @stefanobaghino-da
 
+# KV Participant
 /ledger/      @digital-asset/kv-participant
 /ledger-api/  @digital-asset/kv-participant
 
-/ledger/ledger-api-common/          @digital-asset/kv-committer @digital-asset/kv-participant
-/ledger/ledger-api-domain/          @digital-asset/kv-committer @digital-asset/kv-participant
-/ledger/ledger-api-health/          @digital-asset/kv-committer @digital-asset/kv-participant
-/ledger/ledger-api-test-tool/       @digital-asset/kv-committer @digital-asset/kv-participant
-/ledger/ledger-configuration/       @digital-asset/kv-committer @digital-asset/kv-participant
-/ledger/ledger-offset/              @digital-asset/kv-committer @digital-asset/kv-participant
-/ledger/participant-state/          @digital-asset/kv-committer @digital-asset/kv-participant
-/ledger/participant-state-index/    @digital-asset/kv-committer @digital-asset/kv-participant
-/ledger/participant-state-metrics/  @digital-asset/kv-committer @digital-asset/kv-participant
-/ledger/sandbox/                    @digital-asset/kv-committer @digital-asset/kv-participant
+# Interfaces that are jointly owned
+/ledger-api/grpc-definitions/       @meiersi-da @digital-asset/kv-participant @digital-asset/kv-committer
+/ledger/participant-state/          @meiersi-da @digital-asset/kv-participant @digital-asset/kv-committer
+/ledger/participant-state-metrics/  @meiersi-da @digital-asset/kv-participant @digital-asset/kv-committer
 
+# Owned by KV Participant with KV Committer added for notifications
```

Comment:
Some of the below need to be co-owned as well, such as the below:
* `/ledger/ledger-api-test-tool` (this regularly needs to be modified based on participant state API _implementation_)
* `/ledger/ledger-configuration/`, `/ledger/ledger-offset` (part of participant state API)
We might want to reconsider the module structure for the latter, and e.g. move all the participant state API related modules into a separate subdirectory (such as `/ledger/participant-state-api/...`).